src/app/layout.tsx

```diff
@@ -2,6 +2,8 @@ import type { Metadata } from "next";
 import { Montserrat } from "next/font/google";
 import "./globals.css";
 import BarkAuthProvider from "@/components/bark/bark-auth";
+import RootHeader from "@/components/ui/rootHeader";
+import RootFooter from "@/components/ui/rootFooter";
```

Comment:
(1) nit: Would appreciate kebab-case file names for consistency. (And don't look at the test file names! 😛)

(2) Can we move them into `src/app/_components/{root-header,root-footer}.tsx`? So we can reserve `src/components/ui` for shadcn-ui.